torchao/dtypes/affine_quantized_tensor.py

```diff
@@ -1219,6 +1413,47 @@ def _linear_fp_act_fp8_weight_impl(
     ).reshape(out_shape)
 
 
+def _linear_fp_act_int4_weight_sparse_marlin_check(input_tensor, weight_tensor, bias):
```

Comment:
cc @jcaip now we support registering layout type and these things in a separate file, maybe we can create a folder for all sparse layouts and move these

Reply:
Yeah, that sounds like a good idea.